src/Core/Commander/Providers/WMTS_Provider.js

```diff
@@ -230,6 +247,13 @@ WMTS_Provider.prototype.computeLevelToDownload = function computeLevelToDownload
             lvl));
 };
 
+WMTS_Provider.prototype.tileTextureCount = function tileTextureCount(tile, layer) {
+    computeTileWMTSCoordinates(tile, layer, this.projection);
```

Comment:
there are twice executions of `computeTileWMTSCoordinates` by node? 
in `subdivideNode` and `WMTS_Provider.prototype.executeCommand`